File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        heapq._heapify_max(nums)
        
        while(k > 0):
            x = heapq._heappop_max(nums)
            k-=1
        
        return x

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> maxHeap = new PriorityQueue<>((a,b) -> b - a);

        for(int i = 0; i < nums.length; i++){
            maxHeap.add(nums[i]);
        }
        int x = 0;
        while(k != 0){
            x = maxHeap.poll();
            k--;
        }

        return x;
    }
}